clappr/clappr-ios
# Read offline assets from assetCache

## 🏁  Goal
Read only the available tracks from offline assets to prevent network access during the offline experience.

## ✅  How to test
1. Embed any offline asset in the application (e.g. the sample.movpkg in the test bundle).
2. Turn off the network access.
3. Make sure all the tracks being shown are available offline.
